### PR TITLE
[Microsoft.Azure.Management.ResourceManager] [DeploymentScripts] New Api-version

### PR DESCRIPTION
**Pull request by jorgecotillo**
Updated generated classes to use latest api-version 2020-10-01

### CODE REVIEW

File: sdk/resources/Microsoft.Azure.Management.Resource/src/Generated/Models/AzureCliScript.cs [hunk 1]
```diff
@@ -77,9 +82,11 @@ public AzureCliScript()
...
-            : base(identity, location, id, name, type, tags)
+        public AzureCliScript(string location, System.TimeSpan retentionInterval, string azCliVersion, string id = default(string), string name = default(string), string type = default(string), ManagedServiceIdentity identity = default(ManagedServiceIdentity), IDictionary<string, string> tags = default(IDictionary<string, string>), SystemData systemData = default(SystemData), ContainerConfiguration containerSettings = default(ContainerConfiguration), StorageAccountConfiguration storageAccountSettings = default(StorageAccountConfiguration), string cleanupPreference = default(string), string provisioningState = default(string), ScriptStatus status = default(ScriptStatus), IDictionary<string, object> outputs = default(IDictionary<string, object>), string primaryScriptUri = default(string), IList<string> supportingScriptUris = default(IList<string>), string scriptContent = default(string), string arguments = default(string), IList<EnvironmentVariable> environmentVariables = default(IList<EnvironmentVariable>), string forceUpdateTag = default(string), System.TimeSpan? timeout = default(System.TimeSpan?))
```
Comment (allenjzhang) [inline sdk/resources/Microsoft.Azure.Management.Resource/src/Generated/Models/AzureCliScript.cs:85]: This is a breaking change. Why was the order changed? Can you add related swagger PR link to the PR description?

Comment (jorgecotillo) [reply sdk/resources/Microsoft.Azure.Management.Resource/src/Generated/Models/AzureCliScript.cs:85]: Sure, here's the PR:

https://github.com/Azure/azure-rest-api-specs/pull/10784

This new api-version makes ManagedIdentity an optional argument, if the ManagedIdentity is not passed we use the user's token to create the supporting resources by leveraging ARM OBO token.

**Review by allenjzhang — CHANGES_REQUESTED** [review]
Could you:

 - Check in the metadata TXT file generated when you run `generate.ps1`[(details)](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#sdk-generation-from-updated-spec)
 
Thanks

**Review by JeffreyRichter — APPROVED** [review]

**Comment by jorgecotillo** [comment]
@allenjzhang metadata file has been pushed, please let me know if there's anything else required. Thanks!

**Comment by jorgecotillo** [comment]
Btw, I just removed the dupe Resources_2020-10-01 from AzSDK.RP.props

**Review by allenjzhang — APPROVED** [review]

**Pull request merged**